azure-pipelines-dependency-graph-testing.yml

```diff
@@ -243,13 +243,22 @@ stages:
               pip install  .
             displayName: 'Install dependencies'
 
+          - script: |
+              # Install dependencies
+              pip install pytest-cov pytest-icdiff pytest-mock pytest-azurepipelines invoke
```

Comment:
I'd like to centralize our dependencies into a requirements file. If these are for testing on azure, you can use this: [requirements-dev-test-pipeline.txt](https://github.com/great-expectations/great_expectations/blob/develop/requirements-dev-test-pipeline.txt).
Otherwise [requirements-dev-contrib.txt](https://github.com/great-expectations/great_expectations/blob/develop/requirements-dev-contrib.txt) or [requirements-dev-lite.txt](https://github.com/great-expectations/great_expectations/blob/develop/requirements-dev-lite.txt) may be appropriate.

Reply:
Yeah, good. Call I forgot about these. Will update.

Reply:
Thanks!